### DISCUSSION TITLE
The signature of “Lottie.xcframework” cannot be validated and may have been compromised.

### DISCUSSION THREAD

**Discussion by jesus-mg-ios** [airbnb/lottie-ios#2339] [General]
## Which Version of Lottie are you using?

Lottie 4.4.1
https://github.com/airbnb/lottie-ios/releases/download/4.4.1/Lottie-Xcode-15.2.xcframework.zip

Xcode 15.3
## Expected Behavior 
No error

## Actual Behavior
<img width="247" alt="image" src="https://github.com/airbnb/lottie-ios/assets/85997060/dd14f096-bfad-4cac-9c93-dc7b58ab9205">

**Comment by calda** [comment]
I can't reproduce this issue testing the linked `Lottie-Xcode-15.2.xcframework.zip` file in a sample project using Xcode 15.3. Could you share a sample project that demonstrates this issue?

**Comment by jesus-mg-ios** [comment]
The project was forced to not trust self signed and ends on this "The signature of “Lottie.xcframework” cannot be validated and may have been compromised." 

Maybe we can do something with it: 

<img width="256" alt="image" src="https://github.com/airbnb/lottie-ios/assets/85997060/cb697582-cdc9-4c88-a42a-37623ef992f7">

**Comment by calda** [comment]
We use a self-signed certificate as explained [here](https://github.com/airbnb/lottie-ios#security). That fingerprint is correct and authentic.

> The project was forced to not trust self signed 

How are you configuring this, and why reject self-signed libraries?

**Comment by calda** [comment]
Converting this to a discussion given this is configuration-related

**Comment by daveobersby** [comment]
We've developed an private Cordova plugin that adds the Lottie.xcframework to the build before Xcode is opened and as a result when the framework is imported imported in this manner the self signed certificate is not trusted by Apple following their 2025 requirement changes. Would you consider using an officially signed certificate instead?
